jenkins/compy_next.sh

```diff
@@ -5,4 +5,5 @@ export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
 export CIME_MACHINE=compy
 source $SCRIPTROOT/util/setup_common.sh
 
-$RUNSCRIPT --compiler pgi
+#$RUNSCRIPT --compiler pgi
+$RUNSCRIPT
```

Comment:
You have to add the 2 lines above to switch to intel?

Reply:
intel is default since it is listed first in config_machines.xml, correct? so no need to specify in this script?

Reply:
I think @rljacob is suggesting that you delete the comment. We already have git for looking back in history, so there's no need to leave it in.

Reply:
Oh I see, you commented out the first line.